chapter_deep-learning-computation/use-gpu_vn.md

```diff
@@ -421,7 +423,12 @@ on the command line (or logging it in a NumPy array) will trigger a global inter
 It is much better to allocate memory for logging inside the GPU and only move larger logs.
 -->
 
-*dịch đoạn phía trên*
+* MXNet có thể chỉ định các thiết bị thực hiện việc lưu trữ và tính toán như CPU hay GPU.
+Mặc định, MXNet tạo dữ liệu trên bộ nhớ chính và sử dụng CPU để tính toán.
+* MXNet yêu cầu tất cả dữ liệu đầu vào để tính toán *nằm trên cùng thiết bị*, tức cùng một CPU hoặc cùng một GPU. 
+* Hiệu năng có thể giảm đáng kể nếu di chuyển dữ liệu một cách thiếu cân nhắc.     
+Một lỗi thường gặp là: việc tính toán mất mát cho các minibatch trên GPU rồi trả kết quả lại trên cửa sổ dòng lệnh (hoặc ghi kết quả vào mảng NumPy) sẽ kích hoạt trình quản lý luồng (*Global Interpreter Lock*) làm tất cả GPU dừng hoạt động.
```

Comment:
```suggestion
Một lỗi thường gặp là: việc tính toán mất mát cho các minibatch trên GPU rồi trả kết quả lại trên cửa sổ dòng lệnh (hoặc ghi kết quả vào mảng NumPy) sẽ kích hoạt khóa trình biên dịch cục bộ (*Global Interpreter Lock*) làm tất cả GPU dừng hoạt động.
```

Reply:
Em nghĩ `Global Interpreter Lock` nên dịch theo chức năng hơn là word by word. Với cả sao `global` lại dịch thành `cục bộ` vậy anh?

Reply:
Oops, Mình nhầm. Đã edit.
À. Mình vẫn chưa biết lý do vì sao dịch theo chức năng mà không nên dịch word by word. Tại vì GIL khi dịch ra word by word vẫn có thể hiểu nó làm gì.

Reply:
Em hiểu ở đây `khoá` là danh từ, mang nghĩa `khoá CỦA trình biên dịch toàn cục`, nếu dịch word by word thì khi đọc cả câu từ `khoá` dễ bị hiểu sang động từ: `khoá "trình biên dịch toàn cục" lại`.

Reply:
Thường thì sau động từ sẽ là danh từ. Nhưng nghĩ như bạn cũng đúng vì "kích hoạt khóa <một cái gì đó>" hay được dùng hằng ngày. Ok, mình đồng ý giữ nguyên bản gốc